blockchain/types/tx_internal_data_serializer_test.go

```diff
@@ -287,6 +289,26 @@ func genDynamicFeeTransaction() TxInternalData {
 	return tx
 }
 
+func genSetCodeTransaction() TxInternalData {
+	tx, err := NewTxInternalDataWithMap(TxTypeEthereumSetCode, map[TxValueKeyType]interface{}{
+		TxValueKeyNonce:             nonce,
+		TxValueKeyTo:                to,
+		TxValueKeyAmount:            amount,
+		TxValueKeyGasLimit:          gasLimit,
+		TxValueKeyGasFeeCap:         gasFeeCap,
+		TxValueKeyGasTipCap:         gasTipCap,
+		TxValueKeyData:              []byte("1234"),
+		TxValueKeyAccessList:        accesses,
```

Comment:
Do we need the `accessList` for the test?
I've also check the Ethereum test codes.

https://github.com/ethereum/go-ethereum/pull/30078/files#diff-d78b498ba524953c810b53b2f1e866245d052a063863ed820c9b4852f8762d69R4306-R4314

```
		txdata := &types.SetCodeTx{
			ChainID:   gspec.Config.ChainID.Uint64(),
			Nonce:     0,
			To:        addr1,
			Gas:       500000,
			GasFeeCap: uint256.MustFromBig(newGwei(5)),
			GasTipCap: uint256.NewInt(2),
			AuthList:  []*types.Authorization{auth1, auth2},
		}
```

Reply:
I don't think it matters either.
These can be considered an extension of the existing test, so I think we can include access_list as well.
https://github.com/kaiachain/kaia/blob/178185beb4c576c6efbed3baf2171ae2db8ac599/blockchain/types/tx_internal_data_serializer_test.go#L271-L288